src/Plutarch/Extra/FixedDecimal.hs

```diff
@@ -67,45 +65,14 @@ import qualified PlutusLedgerApi.V1 as PlutusTx
  @since 3.12.0
 -}
 newtype FixedDecimal (exp :: Natural) = FixedDecimal {numerator :: Integer}
-  deriving stock
-    ( -- | @since 3.12.0
-      Generic
-    , -- | @since 3.12.0
-      Eq
-    , -- | @since 3.12.0
-      Ord
-    , -- | @since 3.12.0
-      Show
+  deriving stock (Generic, Eq, Ord, Show)
+  deriving newtype
+    ( -- | @since 3.15.4
+      Aeson.ToJSON
+    , -- | @since 3.15.4
+      Aeson.FromJSON
```

Comment:
I would _strongly_ advise against `newtype` derivations for serialization. I've been bitten by this before, where an (otherwise invisible) representation change caused live code to break. Better to derive via: at least then, we'll know.

Furthermore, this will serialize as a single number, which is a touch misleading. A single-field object would be much better IMHO.